package.json

```diff
@@ -22,11 +22,11 @@
     "@types/jest": "^26.0.22",
     "conditional-type-checks": "^1.0.5",
     "copyfiles": "^2.4.1",
-    "jest": "^26.6.3",
+    "jest": "^29.7.0",
     "prettier": "^2.2.1",
     "rxjs": "^6.6.7",
-    "ts-jest": "^26.5.4",
-    "tslib": "^2.1.0",
-    "typescript": "^4.2"
+    "ts-jest": "^29.1.2",
+    "tslib": "^2.6.2",
+    "typescript": "^5.4"
```

Comment:
Bumping to typescript 5.X allows the use of [const type params](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-0.html#const-type-parameters).